### ISSUE TITLE
Not able to delete the default "admin" user

### ISSUE THREAD

**Issue by ronrem** [ansibleguy76/ansibleforms#240]
**Describe the bug**
The GUI and API do not give the ability to delete or rename the "admin" user.

**To Reproduce**
1. via https://ansibleforms/#/users the delete and update button are disabled, even when logging in via different admin user,
2. a DELETE of https://ansibleforms/api/v1/user/1 give message: "You cannot delete user 'admin'"

**Expected behavior**
Have the ability to delete or rename the "admin" user.

**Version**
ansibleforms v5.0.7

**Deployment**
Deployed ansibleforms with :
- kubernetes

**Comment by ansibleguy76** [comment]
Hi, that's by design, to not lock you out.  However, there is meanwhile a env var BYPASS_ADMIN in case you lock yourself out, you will then automatically be logged in as admin.  Note that in a next release I will add role options.  It will allow you to decide which options are available per role (show_designer, show_logs, show_settings, show_debug_buttons) which will grant you the option to do define your own admin role.  This will be restritive, meaning, if you set show_settings=false on public role (everyone), it will take away the settings for everyone.  Any suggestion on how this should be taccled are more than welcome.  I'm driving solo here.

**Comment by ronrem** [comment]
Thank you for the quick follow-up.

To provide some context for my question: we are using Kubernetes (K8s) and Helm, and I want to configure the credentials for AnsibleForms using a secret stored in Kubernetes. Currently, the database creation process employs a static username and password. I am using this static admin account to create a new user, after which I plan to remove the admin user. It would be more flexible if the database creation process allowed the admin user to be created with a username and password retrieved from environment variables.

The idea of creating roles makes a lot of sense; I’ll take some time to think that through. However, introducing the option to enable or disable the items you mentioned is definitely a great starting point.

**Comment by ansibleguy76** [comment]
Yes, that has been a struggle for me as well.  The credentials are encrypted in the database, creating the default admin user is thus not that obvious.  Simply providing username & password for admin in ENV vars, makes a lot more sense really.  You can never lock yourself out this way.  changing this, wouldn't break.  I would fall back to the env vars if that use doesn't exist in the database.  Good suggestion.  Will see how fast I can implement this.  And i'll only allow database removal if the env vars are present.

**Comment by ronrem** [comment]
Thank you very much!  

Currently, after deployment, the database and schema creation process is initiated by the user upon the application's first launch. We deploy AnsibleForms using Helm and execute the `create_schema_and_tables.sql` MySQL script via a Job template. Could we introduce flexibility in the use of the admin username and password to better accommodate this deployment approach?

**Comment by ansibleguy76** [comment]
in 5.0.8 beta, you can now remove admin user.  a user of choice can now be created (at startup) using ENV vars "ADMIN_USERNAME" and "ADMIN_PASSWORD".

On top of this, you can add options to roles now like (showDesigner, showSettings, showLogs, ...), which allows you to choose your own "admin" roles or create some architect role.  Read the help online about roles
